app/javascript/dashboard/i18n/locale/en/resetPassword.json

```diff
@@ -8,7 +8,7 @@
     },
     "API": {
       "SUCCESS_MESSAGE": "Password reset link has been sent to your email",
-      "ERROR_MESSAGE": "Could not connect to Woot Server, Please try again later"
+      "ERROR_MESSAGE": "Uh ho! We could not find any user with the specified email"
```

Comment:
Should we use what ever is coming from the backend?

Reply:
Yes